02 Toolbar/src/common/managers/textAreaManager.ts

```diff
@@ -0,0 +1,60 @@
+
+// Based on: http://stackoverflow.com/questions/1064089/inserting-a-text-where-cursor-is-using-javascript-jquery
+class TextAreaManager {
```

Comment:
let's call the folder:

./src/common/ui/utils/textArea.ts

and the class name

TextAreaUtil

I know it's not ideal but rather let's avoid blocking this pull because of this.

Do you agree?